packages/variant-theme/src/ThemeGenerator.ts

```diff
@@ -0,0 +1,426 @@
+/* eslint-disable */
+/* tslint-disable */
+//@ts-nocheck
+import {
+  getColorFromString,
+  IColor,
+  updateA
+} from '@fluentui/react';
+import { BrandVariants, Theme } from '@fluentui/react-components';
+import { VariantThemeType } from '@fluentui/scheme-utilities';
+import {
+  black,
+  white,
+  grey,
+  whiteAlpha,
+  blackAlpha,
+  AlphaColors,
+  type Greys,
+} from '@fluentui/react-migration-v8-v9';
+import * as lodash from 'lodash';
+import { contrast, hex_to_sRGB } from './ThemeDesigner/src/colors';
+
+const standardContrastRatio: number = 4.5;
```

Comment:
no need for explicit annotation, ts knows this is `number` from the assigned value

```suggestion
const standardContrastRatio = 4.5;
```

Reply:
fixed